src/utils/index.ts

```diff
@@ -73,3 +73,22 @@ export const constructSlackMessage = (text: string) => {
 		},
 	];
 };
+
+export const sendBotStatusMessage = async (text: string): Promise<Response> => {
+	const resObject = {
+		text,
+		response_type: 'in_channel',
+	};
+
+	const botStatusResp = await fetch(SLACK_WEBHOOK_BOT_STATUS, {
+		method: 'POST',
+		body: JSON.stringify(resObject),
+		headers: { 'Content-Type': 'application/json' },
+	});
+
+	if (botStatusResp.ok) {
+		return botStatusResp;
+	} else {
+		throw new Error('Unable to send bot status message');
```

Comment:
where would this be seen?

Reply:
This gets caught by the catch in shorten.ts. It gets shown the same way as the error message for not including the path or url. However, it would be kinda late tho cuz the link has already been shortened. I was thinking of sending the bot status message before the request to shorten was sent but decided against it cuz I thought it would be worse to say that the link has been shortened when it actually hasn't.

Reply:
By too late, what do you mean? I thought the link shortener sends a 401 for some cases.

Reply:
By too late, I meant that the it's too late to send an error message, and the error message will be misleading. Even though an error was sent to the user, the link was still shortened.